### DISCUSSION TITLE
How to use different work flow to get different material parameter

### DISCUSSION THREAD

**Discussion by JayChang-zhe** [assimp/assimp#4539] [Q&A] [answered]
I was trying to get the different material parameter with Assimp 5.2.3,something like: Metallic, Roughness, Specular etc,  inside the material.h I can see two work flow which is Metallic/Roughness Workflow & Specular/Glossiness Workflow, but when i get the value, it's always different,  or i need to get and calculate?  Don't really know how to get it and how to use those two workflow, is there any doc to get?  Thanks in advance
![image](https://user-images.githubusercontent.com/53027749/169766775-3351356c-49de-4fd3-8762-0b111e0e2c4b.png)

**Comment by kimkulling** [comment, marked as answer]
The workflow is pretty simple:

```
    aiColor4D diffuse = {};
    if (aiGetMaterialColor(material, AI_MATKEY_COLOR_DIFFUSE, &diffuse) == AI_SUCCESS ) {
        setColor4(...);
    }
```
